### DISCUSSION TITLE
Não salvar alterações usando EF

### DISCUSSION THREAD

**Discussion by ccleverton** [balta-io/7194#4] [Perguntas e Respostas]
Salve!

Pessoal, como eu faço para alterar os dados de uma Query do EF (alguns ajustes para meu sped), mas queria descartar algumas alterações. Existe alguam maneira ?

**Comment by andrebaltieri** [comment]
Boa tarde @ccleverton, sua pergunta está muito vaga... posta um pedaço de código, algum erro, algumas coisas que já tentou fazer... assim podemos te ajudar melhor!
